nursery/nursery.go

```diff
@@ -255,6 +255,11 @@ func (nursery *Nursery) registerBlockListener(currency boltz.Currency) *utils.Ch
 }
 
 func (nursery *Nursery) getFeeEstimation(currency boltz.Currency) (float64, error) {
+	if currency == boltz.CurrencyLiquid && nursery.network == boltz.MainNet {
+		if _, ok := nursery.onchain.Liquid.Tx.(*onchain.BoltzTxProvider); ok {
+			return 0.01, nil
```

Comment:
There is an API for that: https://api.boltz.exchange/swagger#/Chain/get_chain__currency__fee